src/sonic-yang-models/yang-models/sonic-events-swss.yang

```diff
@@ -102,11 +102,11 @@ module sonic-events-swss {
             }
 
             leaf used_cnt {
-                type uint8;
+                type uint32;
```

Comment:
How do you konw uint32 is large enough?

Reply:
Changed to uint32 as per defined here: https://github.com/sonic-net/sonic-swss/blob/master/orchagent/crmorch.h#L99